group2.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <title>Group 2</title>
+    <h1>Nicole</h1>
+    <h1>Janet</h1>
+    <h1>Anaidys</h1>
```

Comment:
Hey y'all. I'm from Boston, MA. I'm currently a high school Geometry teacher at KIPP Academy Lynn Collegiate. After 5 years of teaching,  I'm looking to make a career switch into tech.